<ui version="4.0" >
 <class>qt4editsub</class>
 <widget class="QWidget" name="qt4editsub" >
  <property name="enabled" >
   <bool>true</bool>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>344</width>
    <height>312</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Subtitle Editor</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QGroupBox" name="gbTime" >
     <property name="title" >
      <string>Time</string>
     </property>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QLabel" name="textLabel1" >
        <property name="text" >
         <string>&amp;Start</string>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
        <property name="buddy" >
         <cstring>teStart</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTimeEdit" name="teStart" />
      </item>
      <item>
       <widget class="QLabel" name="textLabel1_2" >
        <property name="text" >
         <string>S&amp;top</string>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
        <property name="buddy" >
         <cstring>teStop</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTimeEdit" name="teStop" />
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbFrame" >
     <property name="title" >
      <string>Frame</string>
     </property>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QLabel" name="textLabel1_3" >
        <property name="text" >
         <string>&amp;Start</string>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
        <property name="buddy" >
         <cstring>sbStart</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sbStart" />
      </item>
      <item>
       <widget class="QLabel" name="textLabel1_2_2" >
        <property name="text" >
         <string>S&amp;top</string>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
        <property name="buddy" >
         <cstring>sbStop</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sbStop" />
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QTableWidget" name="tabLines" >
       <column>
        <property name="text" >
         <string>Lines</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QPushButton" name="pbBold" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>1</hsizetype>
           <vsizetype>0</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize" >
          <size>
           <width>40</width>
           <height>32767</height>
          </size>
         </property>
         <property name="font" >
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text" >
          <string>B</string>
         </property>
         <property name="shortcut" >
          <string>Ctrl+H</string>
         </property>
         <property name="checkable" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbItalic" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>0</hsizetype>
           <vsizetype>0</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize" >
          <size>
           <width>40</width>
           <height>32767</height>
          </size>
         </property>
         <property name="font" >
          <font>
           <italic>true</italic>
          </font>
         </property>
         <property name="text" >
          <string>I</string>
         </property>
         <property name="shortcut" >
          <string>Ctrl+H</string>
         </property>
         <property name="checkable" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbUnderline" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>0</hsizetype>
           <vsizetype>0</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize" >
          <size>
           <width>40</width>
           <height>32767</height>
          </size>
         </property>
         <property name="font" >
          <font>
           <underline>true</underline>
          </font>
         </property>
         <property name="text" >
          <string>U</string>
         </property>
         <property name="shortcut" >
          <string>Ctrl+H</string>
         </property>
         <property name="checkable" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbStrikeout" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>1</hsizetype>
           <vsizetype>0</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize" >
          <size>
           <width>40</width>
           <height>32767</height>
          </size>
         </property>
         <property name="font" >
          <font>
           <strikeout>true</strikeout>
          </font>
         </property>
         <property name="text" >
          <string>S</string>
         </property>
         <property name="shortcut" >
          <string/>
         </property>
         <property name="checkable" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QPushButton" name="pbInsertLine" >
       <property name="text" >
        <string>&amp;Insert line</string>
       </property>
       <property name="shortcut" >
        <string>Alt+I</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbDeleteLine" >
       <property name="text" >
        <string>&amp;Delete line</string>
       </property>
       <property name="shortcut" >
        <string>Alt+D</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbCancel" >
       <property name="text" >
        <string>&amp;Cancel</string>
       </property>
       <property name="shortcut" >
        <string>Alt+C</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbOk" >
       <property name="text" >
        <string>&amp;Ok</string>
       </property>
       <property name="shortcut" >
        <string>Alt+O</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources/>
 <connections/>
</ui>
